spec/views/form/page_view_spec.rb

```diff
@@ -48,42 +79,42 @@ def assign_attributes(object, attrs)
     let(:expected_guidance) { /What counts as income?/ }
 
     context "with radio type" do
-      let(:attribs) { { type: "radio", answer_options: { "1": "A", "2": "B" } } }
+      let(:question_attribs) { { type: "radio", answer_options: { "1": "A", "2": "B" } } }
```

Comment:
nitpicky, but `question_attributes` is not that much longer and is more pleasant to read 👀